### PR TITLE
TSS: Add some more functionality to tss layer

### PR DESCRIPTION
**Pull request by ChriMarMe**
Signed-off-by: Christopher Meis <christopher.meis@9elements.com>

### CODE REVIEW

File: pkg/tss/capabilities.go [hunk 1]
```diff
@@ -73,3 +74,64 @@ func readTPM20VendorAttributes(rwc io.ReadWriter) (TPMInfo, error) {
 		FirmwareVersionMinor: int(fw.Value & 0x0000ffff),
 	}, nil
 }
+
+func takeOwnership12(rwc io.ReadWriteCloser, ownerPW, srkPW string) (bool, error) {
+	var ownerAuth [20]byte
```
Comment (zaolin) [inline pkg/tss/capabilities.go:79]: Is this 20 bytes zero?

Comment (ChriMarMe) [reply pkg/tss/capabilities.go:79]: What do you mean? Does it matter here? I use it to store a hash value of sha1 (160bit = 20 byte).
Please elaborate

Comment (zaolin) [reply pkg/tss/capabilities.go:79]: you need to define the WELL_KNOWN_SECRET which is 20 bytes of zero.

Comment (ChriMarMe) [reply pkg/tss/capabilities.go:79]: [20]byte initializes with default of zeros. Should be fine.

**Pull request merged**